lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -146,13 +179,24 @@ function ($handle, $job, $e) {
             function ($handle, $job, $e) {
                 ++$this->jobsCompleted;
                 $this->log(sprintf('Jobs completed: %u', $this->getJobsCompleted()));
+
+                if (arMemprofUtils::getMemprofEnabled()) {
```

Comment:
I think it's better to instantiate an arMemprof class instead of using static methods.  Using standard classes allows the possibility of defining a standard interface and extending it for specific profiling tools, as well as allowing unit testing.